packages/web-components/src/components/va-button/test/va-button.e2e.ts

```diff
@@ -274,30 +274,30 @@ describe('va-button', () => {
     expect(descriptionSpan).not.toBeNull();
     expect(descriptionSpan.textContent).toBe('Button description.');
   });
-});
 
-it(`renders a default submit button variant`, async () => {
```

Comment:
I didn't change anything with these tests. I just moved them into the actual testing block.